server/data/assessmentStore.ts

```diff
@@ -43,6 +43,12 @@ export default class AssessmentStore {
     return this.client.get(key)
   }
 
+  public async deleteAssessment(sessionId: string, nomisId: string, pathway: string) {
```

Comment:
We have used "nomsId" rather than "nomisId" (not sure which is right) in most places so might be worth changing this.